lib/internal/test_runner/harness.js

```diff
@@ -83,7 +83,7 @@ function createTestTree(rootTestOptions, globalOptions) {
   return globalRoot;
 }
 
-function createProcessEventHandler(eventName, rootTest) {
+function createProcessEventHandler(eventName, rootTest, cwd) {
```

Comment:
I would prefer to get the `cwd` from the `rootTest` here instead of adding another argument.

Reply:
done